job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -25,7 +26,7 @@ pipeline{
                 build job: 'nightly/main', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'KIE_GROUP_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-main/content-compressed'],
                         [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: 'main'],
-                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.12.0'],
+                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: "${PROD_VERSION}"],
```

Comment:
@Ginxo shouldn't this be ${PRODUCT_VERSION} ??

Reply:
@mbiarnes you are right